queries/sparql/query-07-drug-to-drug-target.rq

```diff
@@ -0,0 +1,15 @@
+=+ title Get drugs and their human protein targets
+=+ description Get drugs and their human protein targets
+=+ endpoint http://kg-hub-rdf.berkeleybop.io/blazegraph/sparql
+
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix bl: <https://w3id.org/biolink/vocab/>
+SELECT ?drug ?druglab ?target ?targetlab WHERE
+{
+  ?drug ?pred ?target .
+  ?drug bl:category bl:Drug .
+  ?target bl:category bl:Protein; <https://www.example.org/UNKNOWN/ncbi_taxid> "9606"^^xsd:string .
```

Comment:
We really should get rid of taxon ids referenced as strings.. Very bad modelling. Should be:

```
?target bl:category bl:Protein; <https://www.example.org/UNKNOWN/ncbi_taxid> <http://purl.obolibrary.org/obo/NCBITaxon_ 9606> .
```

Reply:
Agreed, but I think this will require a change to KGX